src/pages/workspaces/workspace/analysis/modals/AnalysisModal.js

```diff
@@ -255,7 +255,7 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
           })
         ])]),
         Utils.cond(
-          [currentToolObj?.isNotebook, () => h(IdContainer,
+          [currentToolObj.label === tools.Azure.label || currentToolObj.label === tools.Jupyter.label, () => h(IdContainer,
```

Comment:
breaking this out to a small helper function would remove the need to repeat here and below.

Reply:
I'm realizing there are variables at the top that simplify this expression enough. I'm going to prefer this way since its consistent with @n8landolt 's current PR https://github.com/DataBiosphere/terra-ui/pull/3528/files#diff-ee734d20d673f7e34911023a4e1a7c1c67fc3dbb7faaee153edeaeb1dce0b44eR302